R/InstanceSettings.R

```diff
@@ -15,27 +15,33 @@ InstanceSettings <- R6::R6Class( # nolint object_name_linter
         "owner",
         "name",
         "id",
-        "lnid",
         "schema_str",
         "schema_id",
         "git_repo",
         "keep_artifacts_local",
-        "api_url",
-        "lamindb_version",
-        "storage",
-        "db_scheme",
-        "db_host",
-        "db_port",
-        "db_database",
-        "db_permissions",
-        "db_user_name",
-        "db_user_password"
+        "api_url"
+      )
+      optional_columns <- c(
```

Comment:
Would it be clearer to call these "settings" or something rather than columns?

Reply:
good point -- done!